types/postcss/index.d.ts

```diff
@@ -0,0 +1,34 @@
+declare module 'postcss/lib/lazy-result' {
```

Comment:
Is it possible to use https://github.com/postcss/postcss/blob/master/lib/postcss.d.ts instead?

Reply:
As I see there is no `LazyResult` class in export, so there is no way to use `default` export from postcss. Regarding to [module resolution](https://www.typescriptlang.org/docs/handbook/module-resolution.html) we have to define types ourself